opentelemetry-sdk/src/logs/log_processor_with_async_runtime.rs

```diff
@@ -126,13 +126,13 @@ impl<R: RuntimeChannel> BatchLogProcessor<R> {
         let inner_runtime = runtime.clone();
 
         // Spawn worker process via user-defined spawn function.
-        runtime.spawn(Box::pin(async move {
+        runtime.spawn(async move {
             // Timer will take a reference to the current runtime, so its important we do this within the
             // runtime.spawn()
-            let ticker = inner_runtime
-                .interval(config.scheduled_delay)
+            let ticker = to_interval_stream(inner_runtime.clone(), config.scheduled_delay)
```

Comment:
we might be able to rewrite the batchlogprocessor to be similar to the one that spawns own thread, which do not rely on interval_steam, but use alternatives to achieve similar goal.

Reply:
Is this something that needs to be addressed in this PR?

Reply:
No.